tutorials/quantum_simulation/Spin chain of 3 atoms in XY mode.ipynb

```diff
@@ -34,7 +34,7 @@
   },
```

Comment:
<p>It seems that the outputs are missing in this notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>